lib/airbrake/rack/notice_builder.rb

```diff
@@ -62,7 +62,7 @@ def add_context(notice)
           context[:action] = @controller.action_name
         end
 
-        notice[:context].merge!(@user.to_hash) if @user
+        notice[:context].merge!(@user.data) if @user
```

Comment:
I guess `as_json` would be better because with `to_hash` I made [this](https://github.com/airbrake/airbrake/pull/466/files#diff-44014eb454f65244a394186265a92a2cL64) mistake. I was calling `{}.to_hash` with always returns itself.